zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/CamundaExporter.java

```diff
@@ -200,14 +200,14 @@ public void export(final Record<?> record) {
     }
   }
 
-  private void ensureCachedRecordsLessThanBulkSize() {
+  private void ensureCachedRecordsLessThanBulkSize(final Record<?> record) {
     final var maxCachedRecords = configuration.getBulk().getSize();
 
     if (writer.getBatchSize() > maxCachedRecords) {
       final var errMsg =
           String.format(
-              "Reached the max bulk size amount of cached records [%d] while waiting for importers to finish, restarting exporter",
-              maxCachedRecords);
+              "Reached the max bulk size amount of cached records [%d] while waiting for importers to finish, retrying export for [%s]",
+              maxCachedRecords, record.toString());
```

Comment:
🔧 this is kind of a nitpicking: but I'm not sure if we should log the whole record. if we need to point to it we shall log the record position.